kippenhahn/kippenhahn.py

```diff
@@ -628,14 +647,17 @@ def CheckParameters(self):
         if not (self._param['Xaxis'] in ['model_number', 'star_age', 'inv_star_age', 'log_model_number', 'log_star_age',
                 'log_inv_star_age']):
             raise ValueError(self._param['Xaxis']+"not a valid option for parameter Xaxis")
-        if not (self._param['Variable'] in ['eps_nuc', 'velocity', 'entropy', 'total_energy', 'j_rot', 'eps_recombination'
-                , 'ionization_energy', 'energy', 'potential_plus_kinetic', 'extra_heat', 'v_div_vesc',
-                'v_div_csound',    'pressure', 'temperature', 'density', 'tau', 'opacity', 'gamma1', 'gamma3', 'dq',
-                'L_div_Ledd', 'Lrad_div_Ledd', 't_thermal', 't_dynamical', 't_dynamical_down', 't_thermal_div_t_dynamical',
-                 'omega_div_omega_crit', 'omega','super_ad', 'vconv', 'vconv_div_vesc', 'conv_vel_div_csound',
-                 't_thermal_div_t_expansion','E_kinetic_div_E_thermal','total_energy_plus_vconv2']):
-            raise ValueError(self._param['Variable']+"not a valid option for parameter Variable")
-
+        
+        """
```

Comment:
Why do you start a second docstring here? Btw. never use docstrings to multi line comments, this is very dirty coding!